ci.sh

```diff
@@ -269,6 +269,10 @@ case "$cmd" in
       cache_download barretenberg-bench-results-$bb_hash.tar.gz
     fi
 
+    # noir-protocol-circuits benchmarks.
+    noir-projects/noir-protocol-circuits/bootstrap.sh bench
+    mv noir-projects/noir-protocol-circuits/bench-out/* ./bench-out/
+
```

Comment:
These benchmarks need to be run on gh now. This is fine as the benchmarks are not timing based but it'll still slow down the collection of data.